### PR TITLE
Add support of Symfony 3.0

### PR DESCRIPTION
**Pull request by francoispluchino**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #105
| License       | MIT
| Doc PR        | ~

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,13 +11,13 @@
     }
   ],
   "require": {
-    "symfony/framework-bundle": "~2.1",
+    "symfony/framework-bundle": "~2.1|~3.0",
     "willdurand/geocoder": "~3.1",
```
Comment (willdurand) [inline composer.json]: what about bumping this version to `~3.3`

Comment (francoispluchino) [inline composer.json]: Do you want to update `willdurand/geocoder` to 3.3 minimum?

Comment (willdurand) [inline composer.json]: mmh nevermind, I thought we add a dependency to a Symfony component, but that's not the case..

**Comment by willdurand** [comment]
thanks

**Pull request merged**